packages/playwright-core/src/server/trace/recorder/snapshotterInjected.ts

```diff
@@ -419,6 +419,8 @@ export function frameSnapshotStreamer(snapshotStreamer: string) {
               continue;
             if (nodeName === 'IFRAME' && name === 'src')
               continue;
+            if (nodeName === 'FRAME' && name === 'src')
```

Comment:
Let's combine this with previous check like `(nodeName === 'IFRAME' || nodeName === 'FRAME')`. If you rebase on top of the fresh "main" branch, that will make more sense.